Lib/multiprocessing/resource_tracker.py

```diff
@@ -237,3 +241,5 @@ def main(fd):
                         warnings.warn('resource_tracker: %r: %s' % (name, e))
                 finally:
                     pass
+
+        sys.exit(exit_code)
```

Comment:
Does it currently exit? Or is that a change in behaviour?

CC @vstinner .

Reply:
@iritkatriel 

It is in the end of the `main(fd)` method, so it exit but normally without specifiying an exit code